packages/menu/src/Menu.ts

```diff
@@ -287,18 +331,48 @@ export class Menu extends SizedMixin(SpectrumElement, { noDefaultSize: true }) {
     }
 
     private async removeChildItem(item: MenuItem): Promise<void> {
+        if (item.focused || item.hasAttribute('focused') || item.active) {
```

Comment:
item is losing `focused` property on blur before it disconnects, but retains attribute  for some reason